File: app/src/main/java/com/liyu/huahui/utils/NetworkUtil.java
package com.liyu.huahui.utils;

import android.content.Context;
import android.net.ConnectivityManager;
import android.net.NetworkInfo;

import com.liyu.huahui.App;

/**
 * copied from https://github.com/Blankj/AndroidUtilCode
 * Thanks
 * Created by liyu on 2017/7/11.
 */

public class NetworkUtil {

    public static boolean isConnected() {
        NetworkInfo info = getActiveNetworkInfo();
        return info != null && info.isConnected();
    }

    private static NetworkInfo getActiveNetworkInfo() {
        return ((ConnectivityManager) App.getContext().getSystemService(Context.CONNECTIVITY_SERVICE)).getActiveNetworkInfo();
    }
}

File: app/src/main/java/com/liyu/huahui/utils/NetworkUtil.kt
package com.liyu.huahui.utils

import android.content.Context
import android.net.ConnectivityManager

/**
 * copied from https://github.com/Blankj/AndroidUtilCode
 * Thanks
 * Created by liyu on 2017/7/11.
 */
object NetworkUtil {
    @JvmStatic
    fun isConnected(ctx:Context): Boolean {
            val info = (ctx.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager).activeNetworkInfo
            return info != null && info.isConnected
        }
}